oaknational/Oak-Web-Application
# Feat/ctc 40/update lesson listing query

## Description

Music year: 2020

### ACs

- [x]  new mv is used in query
- [x]  schema is updated
- [x]  overrides and exceptions are implemented
- [x]  navigation is uneffected by changes

## Issue(s)

Fixes #

## How to test

1. Go to https://deploy-preview-3001--oak-web-application.netlify.thenational.academy
2. Click on lesson listing
3. You should see no regressins 

## Screenshots

How it used to look (delete if n/a):
{screenshots}

How it should now look:
{screenshots}

## Checklist
